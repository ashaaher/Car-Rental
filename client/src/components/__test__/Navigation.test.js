import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Navigation from '../Navigation';

it('Navigation Page matches snapshot', () => {
  const wrapper = shallow(<Navigation />);
  expect(wrapper).toMatchSnapshot();
})