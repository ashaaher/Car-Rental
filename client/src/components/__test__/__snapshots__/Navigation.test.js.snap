// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Page matches snapshot 1`] = `ShallowWrapper {}`;
